proposals/PROPOSALS.md

```diff
@@ -1,141 +1,126 @@
+ 
+The committee will publicize the proposal in a variety of Haskell fora, as they deem appropriate for the contents of the proposal.
+At the very least, all proposals will be announced on Discourse.
+
+### Community Discussion and Revision
+
+The next step is community discussion.
+During community discussion, the committee has two roles: to constructively contribute technical insights, and to keep the discussion on track.
+By "on track", we mean that discussion is relevant, respectful, and productive.
+
+Discussing alternatives to the proposal or parts of it that may better achieve its goals is relevant, as is questioning whether the goals are in fact beneficial to the community. Promoting alternative goals or proposing ways to reach them is not. In other words, comments like "you should solve this other problem instead" are not encouraged, while comments like "why not use this other library to achieve the goals?" are.
+
+Respectful discussion lives up to the [respectful communication guidelines](https://haskell.foundation/guidelines-for-respectful-communication/). Additionally, we must take special effort to listen to each others' experiences and take them seriously. Haskell is used in a variety of contexts with a variety of cultures, interests, and requirements, and it's important that we are able to understand these needs, even if we end up not being able to serve all of them. The needs of researchers, business application developers, and open-source contributors are all important, as are those of people in different industries, different countries, and with different backgrounds.
+
+Productive discussion contributes new insights as it progresses, and doesn't go in circles. When a point has been made, there's usually no need to make it again, although it can be worth checking whether a particular need or interest is unique or shared.
+
+From time to time, the committee will summarize the current state of knowledge in the discussion thread. When there is no longer productive discussion, the committee will proceed to the next step, recommendation.
+
+### Recommendation
+
+The final step of a proposal is for the committee to give a recommendation.
+While the committee should take community discussion into account, they are expected to make recommendations based on their own knowledge, rather than simply reflecting the voices of the participants in the discussion.
+Fundamentally, there are three potential recommendations: rejection, revision, and acceptance.
+*Rejection* means that the committee does not believe that the proposal would, in the balance, bring value to the community, and that it should not be acted upon.
+Furthermore, rejection implies that the committee sees no route to making the proposal acceptable that is short of rewriting it from scratch.
+Rejection may be because the problem statement itself is considered to be not one that should be solved, or that the harms of the proposed solution would be greater than the benefits.
```

Comment:
What about "just not worth the expenditure of resources?" We should be able to reject stuff that's just... meh.

Reply:
If I rephrased to "or that the benefits would not be sufficiently larger than the costs and harms of the proposed solution", would that address your concern?

Reply:
Thanks I updated the text to your suggestion.